### PR TITLE
#168781617 - Set Up scss

### PR DESCRIPTION
**Pull request by Cheza-Dzabala**
#### What does this PR do?
- Add Scss processing to Barefoot Nomad

#### Description of Task to be completed?
- Add Scss Processing ability to the react application

#### How should this be manually tested?

1. Clone this branch.
2. Run ` npm install `
3. Run ` npm start `

#### Any background context you want to provide?
N / A

#### What are the relevant pivotal tracker stories?
#168781617

Screenshots (if appropriate)
Screen Shot 2019-11-26 at 14 00 46
Screen Shot 2019-11-26 at 14 00 52
Screen Shot 2019-11-26 at 14 00 58
Screen Shot 2019-11-26 at 14 03 39

### CODE REVIEW

**Review by sabin18 — APPROVED** [review]
good job @Cheza-Dzabala

**Review by nignanthomas — APPROVED** [review]
LGTM @Cheza-Dzabala

File: src/styles/colors.scss [hunk 1]
```diff
@@ -0,0 +1,2 @@
+$primary: #073763;
```
Comment (nakiwuge) [inline src/styles/colors.scss]: kindly rename this file to variables so that it can take all variables.

Comment (Cheza-Dzabala) [reply src/styles/colors.scss]: Noted

**Pull request closed without merging**